cfme/tests/v2v/test_v2v.py

```diff
@@ -1,9 +1,96 @@
+from cfme.utils.log import logger
+from cfme.utils.update import update
 
-pytestmark = [pytest.mark.ignore_stream('5.8', '5.9')]
+pytestmark = [
+    pytest.mark.ignore_stream('5.8', '5.9'),
+    pytest.mark.meta(server_roles=['+git_owner']),
+    pytest.mark.provider([VMwareProvider, RHEVMProvider], scope='module')
+]
+
+
+@pytest.yield_fixture(scope='module')
+def get_two_providers(appliance, request, provider):
+    # TODO: Remove this tweak by solution to add two providers in single session
+    x, y = request.session.items
+    prov1 = str(x).split('[')[1].split(']')[0]
+    prov2 = str(y).split('[')[1].split(']')[0]
+
+    pro1 = providers.get_crud(prov1)
+    pro2 = providers.get_crud(prov2)
+
+    if pro1.one_of(RHEVMProvider):
+        rhvm_prov = pro1
+        nvc_prov = pro2
+    else:
+        rhvm_prov = pro2
+        nvc_prov = pro1
+    yield (rhvm_prov, nvc_prov)
+
+
+@pytest.yield_fixture(scope='module')
+def initial_setup(appliance, request, get_two_providers):
+    """
+    Steps:
+        1.Add rhvm and nvc providers
+        2.Validate host credentials
+        3.Assign tags to conversation host
+        4.Import automate domain via github repo
+    """
+    rhvm_prov, nvc_prov = get_two_providers
+
+    # Validating host credentials
+    test_host = random.choice(rhvm_prov.data["hosts"])
+    for host_obj in conf.cfme_data.get('management_systems', {})[rhvm_prov.key].get('hosts', []):
+        if test_host.name == host_obj['name']:
+            host_data = host_obj
+    host_collection = appliance.collections.hosts
+    host_obj = host_collection.instantiate(name=test_host.name, provider=rhvm_prov)
+    with update(host_obj, validate_credentials=True):
+        host_obj.credentials = host.get_credentials_from_config(host_data['credentials'])
+
+    # Assigning tags to conversation host
+    tag1 = Tag(display_name='t', category=Category(display_name='V2V - Transformation Host *'))
+    tag2 = Tag(display_name='VDDK', category=Category(display_name='V2V - Transformation Method'))
+    host_obj.add_tags(tags=(tag1, tag2))
+
+    # Importing automate domain via github repo
+    repo = AutomateGitRepository(url="https://github.com/fdupont-redhat/v2v-automate.git",
+                                 verify_ssl=True,
+                                 appliance=appliance)
+    domain = repo.import_domain_from(**{'branch': 'origin/extended_reporting_and_rhv_upload'})
+    request.addfinalizer(domain.delete_if_exists)
+
+
+vms = []
+
+
+@pytest.fixture(scope="module")
+def vm_list(get_two_providers):
+    """Fixture to provider vm list for migration"""
+    rhvm_prov, nvc_prov = get_two_providers
+    # TODO: add template names and configuration in yaml and remove following code
+    templates = [nvc_prov.data.templates.small_template['name']]
+    for template in templates:
+        vm_name = random_vm_name(context='v2v')
+        collection = nvc_prov.appliance.provider_based_collection(nvc_prov)
```

Comment:
take `appliance` directly like
`def vm_list(appliance, get_two_providers):`
and use it `collection = appliance.provider_based_collection(nvc_prov)`